internal/criocli/criocli.go

```diff
@@ -808,6 +808,14 @@ func getCrioFlags(defConf *libconfig.Config) []cli.Flag {
 			EnvVars:     []string{"CONTAINER_REGISTRIES_CONF"},
 			TakesFile:   true,
 		},
+		&cli.StringFlag{
+			Name:        "registries-conf-dir",
+			Usage:       "path to the registries.conf.d directory",
+			Destination: &defConf.SystemContext.SystemRegistriesConfDirPath,
+			Hidden:      true,
+			EnvVars:     []string{"CONTAINER_REGISTRIES_CONF_D"},
```

Comment:
can this be `CONTAINERS_REGISTRIES_CONF_DIR` to be consistent with the other fields that take a directory?